azimuth/utils/plots.py

```diff
@@ -46,6 +48,14 @@ class Colors(str, Enum):
     Text = "#2a3d40"
 
 
+COLORS_PER_SPLIT = {DatasetSplitName.eval: Colors.DataViz1, DatasetSplitName.train: Colors.DataViz2}
+
+SPLIT_PRETTY_NAMES = {
```

Comment:
Oh I like the "arrangement" you used for `SPLIT_PRETTY_NAMES`. I think it could be applied to `SPLIT_COLORS` too. I think we could even go super explicit with `DATASET_SPLIT_COLORS`, and `DATASET_SPLIT_PRETTY_NAMES` (I have that exact same name in the front end), especially since "split" could be a verb without the context of "dataset split". What do you think?
```suggestion
DATASET_SPLIT_COLORS = {DatasetSplitName.eval: Colors.DataViz1, DatasetSplitName.train: Colors.DataViz2}

DATASET_SPLIT_PRETTY_NAMES = {
```